### DISCUSSION TITLE
Unable to retrieve available IP address from API with ip prefix

### DISCUSSION THREAD

**Discussion by malo0301** [netbox-community/netbox#13226] [Help Wanted!]
### NetBox version

v3.4.8

### Python version

3.8

### Steps to Reproduce

1. Go to https://**netbox.domain.com**/ipam/prefixes/**PREFIX_ID**/ip-addresses/
2. Modify url to switch in api mode: https://**netbox.domain.com**/**api**/ipam/prefixes/**PREFIX_ID**/ip-addresses/
3. Get error message: **The requested page does not exist.** 

### Expected Behavior

I would like to get list of used AND available ip adress

### Observed Behavior

An error message is display: The requested page does not exist.

**Comment by twrona** [comment]
HI, I have the same problem, :( I think it's not possible. Maybe we need to ask a feature ?

**Comment by bitcollector1** [comment]
You can see the addresses from the Prefix itself, not sure if this is what you want 

https://netdevbox.me.com/api/ipam/ip-addresses/?parent=10.254.84.0/22

**Comment by abhi1693** [comment]
APIs are not 1-1 mapping of the UI URL. They may seem similar but the endpoints are maintained separately. I do not have a 3.4.x NetBox handy with my but you should look in the swagger schema to find the relevant API. From NetBox v3.5.6, this is what you need https://demo.netbox.dev/api/schema/swagger-ui/#/ipam/ipam_prefixes_available_ips_list

**Comment by bitcollector1** [comment]
Just found out that Swagger is missing the "Token" for the CURL when it presents the command to be used. 

Swagger presents this as the CURL to be used: 
```
curl -X 'GET'   'https://netdevbox.me.com/api/ipam/prefixes/181/available-ips/'   -H 'accept: application/json'   -H 'Authorization: 0ee9654cccaa1d17fabb2f689c3587262300163d'
```

Token should be added in order for the curl command to actually work as expected. 
```
curl -X 'GET'   'https://netdevbox.me.com/api/ipam/prefixes/181/available-ips/'   -H 'accept: application/json'   -H 'Authorization: Token 0ee9654cccaa1d17fabb2f689c3587262300163d'
```
